strapi/strapi
# fix(content-releases): change the submit button action name on edit

### What does it do?

It changes the Release Modal submit button label from "Continue" to "Save". It changes the logic to enable/disable the button, we enable the button on edit only when something is changed in the Release name and it is not empty

### Why is it needed?

If there is no change in the edit release action, button should be disabled and the button action name should be “Save” instead of “Continue”, currently instead we show always Continue and the button is enabled if we add content in the input text

### How to test it?

- Step 1: Go to the Release Details Page
- Step 2: Click on the 3 dots at the top and click on Edit
- Step 3: See that the “Continue” button
